rpc/src/v1/helpers/dispatch.rs

```diff
@@ -474,7 +474,7 @@ pub fn execute<D: Dispatcher + 'static>(
 					.map(ConfirmationResponse::SignTransaction)
 				).boxed()
 		},
-		ConfirmationPayload::Signature(address, mut data) => {
+		ConfirmationPayload::PostSignTransaction(address, mut data) => {
```

Comment:
It does return it in a pretty specific way though, not sure what that should be called: https://github.com/paritytech/parity/blob/64a724092ebda0bea5b1f8e6a60ec6d0fe4869ec/rpc/src/v1/helpers/dispatch.rs#L477